coalib/bearlib/languages/documentation/default.coalang

```diff
@@ -1,5 +1,7 @@
 [PYTHON]
-doc-marker = """, , """
```

Comment:
You have to change the tests too:

https://github.com/coala/coala/blob/1db00d706353bb0d8a175c6d561df21e51cad6e7/tests/bearlib/languages/documentation/DocstyleDefinitionTest.py#L137

https://github.com/coala/coala/blob/1db00d706353bb0d8a175c6d561df21e51cad6e7/tests/bearlib/languages/documentation/DocBaseClassTest.py#L269-L274

Reply:
I know @GabrielDS I pushed it early for some reason. :)

Reply:
:smile: